Userland/Libraries/LibJS/AST.cpp

```diff
@@ -1475,25 +1389,22 @@ ThrowCompletionOr<ClassElement::ClassValue> StaticInitializer::class_element_eva
     return ClassValue { normal_completion(body_function) };
 }
 
-Value ClassExpression::execute(Interpreter& interpreter, GlobalObject& global_object) const
+Completion ClassExpression::execute(Interpreter& interpreter, GlobalObject& global_object) const
 {
     InterpreterNodeScope node_scope { interpreter, *this };
 
     // FIXME: Set value.[[SourceText]] to the source text matched by ClassExpression.
-    return TRY_OR_DISCARD(class_definition_evaluation(interpreter, global_object, m_name, m_name.is_null() ? "" : m_name));
+    auto value = TRY(class_definition_evaluation(interpreter, global_object, m_name, m_name.is_null() ? "" : m_name));
+    return value;
```

Comment:
Can we not directly put the TRY in the return here or does that break because of the ThrowCompletionOr<>?

Reply:
Ah, this is because I first had the spec comments in the same patch but later removed all those lines, so this remained. Will update, but it becomes two separate lines in that commit again.